Do not install /etc/header_postinst.d/dkms, at this moment Debian kernel doesn't support it.
--- a/Makefile
+++ b/Makefile
@@ -70,8 +70,8 @@ install-doc:
 	install -p -m 0644 $(DOCFILES) $(DOCDIR)
 
 install-debian: install install-doc
-	mkdir   -p -m 0755 $(KCONF)/header_postinst.d
-	install -p -m 0755 kernel_postinst.d_dkms $(KCONF)/header_postinst.d/dkms
+	#mkdir   -p -m 0755 $(KCONF)/header_postinst.d
+	#install -p -m 0755 kernel_postinst.d_dkms $(KCONF)/header_postinst.d/dkms
 	mkdir   -p -m 0755 $(ETC)/template-dkms-mkdeb/debian
 	ln -s template-dkms-mkdeb $(ETC)/template-dkms-mkdsc
 	install -p -m 0664 template-dkms-mkdeb/Makefile $(ETC)/template-dkms-mkdeb/
